src/stripeWorkspaceState.ts

```diff
@@ -58,3 +64,25 @@ export function retrieveEventDetails(extensionContext: vscode.ExtensionContext,
 export function clearEventDetails(extensionContext: vscode.ExtensionContext) {
   extensionContext.workspaceState.update(eventDetailsKey, new Map<string, any>());
 }
+
+export function getWebhookEndpoint(extensionContext: vscode.ExtensionContext) {
+  return extensionContext.workspaceState.get(webhookEndpointKey, null);
```

Comment:
nit: you can also just not pass in a default value as well: `extensionContext.workspaceState.get(webhookEndpointKey);`

Reply:
Updated, ptal @gracegoo-stripe